docs/elastic_inference.md

```diff
@@ -0,0 +1,182 @@
+# Model Serving with Amazon Elastic Inference 
+
+## Contents of this Document
+* [Introduction](#introduction)
+* [Custom Service](#custom-service)
+* [Creating a EC2 instance with EIA support](#creating-a-ec2-instance-with-eia-support)
+* [Custom Service file with EIA](#custom-service-file-with-eia)
+* [Running elastic inference on a resnet-152](#running-elastic-inference-on-a-resnet-152)
+## Introduction
+
+Amazon Elastic Inference allows you to attach low-cost GPU-powered acceleration to Amazon EC2 and Amazon SageMaker instances to reduce the cost of running deep learning inference by up to 75%. With MMS it is easy to deploy an MXNet based model, taking advantage of the attacble hardware accelerator.
+In this document we explore using Elastic Inference Acclerator (EIA) attached to a Compute Optimized EC2 instance.
+
+## Custom Service
+
+The capability to run model inference with the Elastic Inference Accelerator (EIA) can be achieved by building the custom service to use the EIA context rather than a GPU or CPU context. An MXNet version with support for EIA is required.
+
+To understand the basics of writing a custom service file refer to the [Custom Service Documentation](https://github.com/awslabs/mxnet-model-server/blob/master/docs/custom_service.md).
+
+## Creating a EC2 instance with EIA support 
+
+To Create an EC2 instance with EIA support there are few pre-requisites. These include:
+1. Configuring a Secuirty Group
```

Comment:
```suggestion
 1. Configuring a Security Group
```

Reply:
You could provide the direct links to these steps like I've done here in the [DLAMI IE prereq docs](https://docs.aws.amazon.com/dlami/latest/devguide/ei-prerequisites.html).